runtime/doc/lsp.txt

```diff
@@ -869,12 +869,14 @@ start({config}, {opts})                                      *vim.lsp.start()*
                   |vim.lsp.ClientConfig|.
       • {opts}    (`table?`) Optional keyword arguments
                   • {reuse_client}
-                    (`fun(client: vim.lsp.Client, config: table): boolean`)
+                    (`fun(client: vim.lsp.Client, config: vim.lsp.ClientConfig): boolean`)
                     Predicate used to decide if a client should be re-used.
                     Used on all running clients. The default implementation
                     re-uses a client if name and root_dir matches.
                   • {bufnr} (`integer`) Buffer handle to attach to if starting
                     or re-using a client (0 for current).
+                  • {quiet} (`boolean`) Suppress error reporting if the LSP
+                    server fails to start (default false).
```

Comment:
`git grep quiet -- ':/*.lua' suggests that we haven't used this name anywhere yet. `silent` would be the conventional name.
```suggestion
                  • {silent} (`boolean`) Suppress error reporting if the LSP
                    server fails to start (default false).
```